ext/libxml/tests/libxml_entity_loading_disabled_by_default.phpt

```diff
@@ -0,0 +1,53 @@
+--TEST--
+libxml_disable_entity_loader()
+--SKIPIF--
+<?php
+if (!extension_loaded('libxml')) die('skip libxml extension not available');
+if (!extension_loaded('dom')) die('skip dom extension not available');
+if (defined('PHP_WINDOWS_VERSION_MAJOR')) die('skip not for Windows'); ?>
```

Comment:
Why is this test not for Windows? Could it be modified (or split) to test the functionality on Windows as well?

Reply:
I don't know to be honest, I just copied another libxml unit test and used it to create a new one.

Reply:
Ah, thanks! I just enabled that other test on Windows (24495ba7780b0656a753678c6337d7a479d35ed3); the same could be applied to this test case.

Reply:
I can rebase and apply the same changes to my test tonight.

Reply:
Done!